third_party/libsuseconnect/libsuseconnect.go

```diff
@@ -110,8 +113,14 @@ func credentials(path *C.char) *C.char {
 
 //export create_credentials_file
 func create_credentials_file(login, password, token, path *C.char) *C.char {
+	credPath := C.GoString(path)
+
+	if !filepath.IsAbs(credPath) {
+		credPath = filepath.Join(cred.DefaultCredentialsDir, credPath)
```

Comment:
why not use the helpers in `cred` ?

```suggestion
		credPath = cred.SystemCredentialsPath(credPath)
```

Reply:
I guess that's what's happening nowadays, `fsRoot` is assumed to `/`, or am I missing a piece of context how the `.so` is used? :sweat_smile:

Reply:
You can change fsRoot to anything in suseconnect with `--root` but the library has no way to alter `fsRoot`.

Reply:
`SystemCredentialsPath` only computes the global credential file `SCCcredentials` from filesystem root. I think there might be cases where `credPath` points to a different creds file. @felixsch might corroborate this better

Reply:
I guess it wouldn't hurt to hardcode the *.so to `/` so it's explicit.